pyspecProcScripts/load_data.py

```diff
@@ -660,22 +660,39 @@ def proc_spincore_generalproc_v1(
     s, direct="t2", include_tau_sub=True, fl=None
 ):
     s.run(np.conj)  # SC flips data in a weird way, this
-    #                                  corrects for that
+    #                 corrects for that
     if include_tau_sub:
         if "tau_us" in s.get_prop("acq_params").keys():
             s[direct] -= s.get_prop("acq_params")["tau_us"] * 1e-6
     s.ft(direct, shift=True)
     for j in [k for k in s.dimlabels if k.startswith("ph")]:
+        # TODO ☐: I wrote the message below, then I realized that you
+        # were doing this in the wrong place!
+        s[j] = ((-s[j] + 4) % 4) / 4 # when we take the complex conjugate,
+        #                              that changes the phase of the
+        #                              phase cycle, as well, so we have
+        #                              to re-label the axis coordinates
+        #                              for the phase cycle to the
+        #                              negative of what they were
+        #                              before.  To keep things sane, we
+        #                              also apply phase wrapping to get
+        #                              positive numbers.
+        s.sort(j)
```

Comment:
please note this change.  I think this should work, but I haven't tested it.  Probably makes sense to read the message below, then this one.

Reply:
no need to divide by 4 because we are already in cyc